<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>891</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widGCalendario" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>721</width>
      <height>471</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="EventosLabel">
    <property name="geometry">
     <rect>
      <x>742</x>
      <y>20</y>
      <width>111</width>
      <height>461</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>891</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Archivo">
    <property name="title">
     <string>[Archivo]</string>
    </property>
    <addaction name="action_Exportar"/>
   </widget>
   <addaction name="menu_Archivo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Exportar">
   <property name="text">
    <string>&amp;Exportar</string>
   </property>
   <property name="toolTip">
    <string>Exportar a formato PDF</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
